docs/application-connector/docs/037-gs-register-secured-api.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Register a secured API
+type: Getting Started
+---
+
```

Comment:
[GS template](https://github.com/kyma-project/community/blob/master/guidelines/templates/resources/getting-started.md)?

Reply:
Right, this will work better as a "details" doc.